server/lib/notifications/notification-service.ts

```diff
@@ -45,8 +49,8 @@ export default class NotificationService {
   }
 
   /**
-   * Adds a new notification with the provided data to the DB, and notifies all clients to update
-   * their local lists.
+   * Creates a new notification for a particular user with the provided data to the DB and notifies
```

Comment:
nit: this sentence doesn't really make sense. I think what you're trying to say is:

> Creates a new notification for a particular user, saves it to the database, and notifies all of the user's connected clients.